content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -104,6 +104,10 @@ Right after `kubeadm init` there should not be any pods in these states.
   likely that the Pod Network solution that you installed is somehow broken. You
   might have to grant it more RBAC privileges or use a newer version. Please file
   an issue in the Pod Network providers' issue tracker and get the issue triaged there.
+- If you install a version of Docker older than 1.12.1, remove the `MountFlags=slave` option
+  when booting `dockerd` with `systemd` and restart `docker`. You can see the MountFlags in `/usr/lib/systemd/system/docker.service`.
+  MountFlags can interfere with volumes mounted by Kubernetes, and put the Pods in `CrashLoopBackOff` state.
+  The error happens when Kubernetes does not find var/run/secrets/kubernetes.io/serviceaccount files.
```

Comment:
It'd be good to quote `var/run/secrets/kubernetes.io/serviceaccount` with backticks (`)

Reply:
fixed it and thanks